syntax = "proto3";

package tc_protobuf;

message RoutingSettingsProto {
    uint32 bus_wait_time = 1;
    double bus_velocity = 2;
}